### ISSUE TITLE
Epic: Menu improvements and header bar removal

### ISSUE THREAD

**Issue by manuelecarlini** [rancher-sandbox/rancher-desktop#4848]
### Problem Description

The navigation menu could be improved by adjusting the spacing.
Furthermore, we agreed that the header bar could be removed in order to have more horizontal space, therefore the logo should be removed and the preference button should be rearranged inside the side menu

### Proposed Solution

Here's the list of modifications to adjust the side menu:
- [x] #5122
- [x] #5592
- [ ] #4850
- [x] #4851
- [x] #5166
- [x] #5167

<img width="772" alt="Screenshot 2023-07-14 at 16 27 23" src="https://github.com/rancher-sandbox/rancher-desktop/assets/6459624/709010a8-264b-4626-ae9e-518767ca8e2c">

In a second moment, we should move the logo and the preference button in the sidebar and remove the header 
<img width="769" alt="Screenshot 2023-07-14 at 16 27 43" src="https://github.com/rancher-sandbox/rancher-desktop/assets/6459624/74c53777-a53e-4221-8e6c-311e589d8ca1">

**Comment by rak-phillip** [comment]
@kwwii and I discussed this Epic. We will not be taking on these changes for v1.10 and will be evaluating how they fit in with tighter `@rancher/shell` integration.

@jandubois @gaktive

**Comment by jandubois** [comment]
> We will not be taking on these changes for v1.10

Thanks, feel free to move to the v1.11 milestone, or remove the milestone altogether until we are ready to schedule it.

**Comment by jandubois** [comment]
I've updated the epic with the following changes:

* Added #5122 because the progress bar needs to move out of the bottom left corner before we can get rid of the header bar.
* Replaced #4849 with #5592 to move the logo to the Welcome page and get rid of the "Rancher Desktop" string in the header.

We moved the epic to the 1.12 milestone when we realized that #5122 was a prerequisite for everything else.
